examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,246 @@ Documentation for other releases can be found at
-
-You can take a look at the logs for a pod by using `kubectl.sh log`.  For example:
+Use a [Secret](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/secrets.md) object to store the
+MySQL password. First create a temporary file called `password.txt`
+and save your password in it. Make sure to not have a trailing newline
+at the end of the password. Then, create the Secret object.
 
-```sh
-$ kubectl logs mysql
+```shell
+kubectl create secret generic mysql-pass --from-file=password.txt
 ```
 
-If you want to do deeper troubleshooting, e.g. if it seems a container is not staying up, you can also ssh in to the node that a pod is running on.  There, you can run `sudo -s`, then `docker ps -a` to see all the containers.  You can then inspect the logs of containers that have exited, via `docker logs <container_id>`.  (You can also find some relevant logs under `/var/log`, e.g. `docker.log` and `kubelet.log`).
-
-### Start the Mysql service
+This secret is referenced by the MySQL and WordPress pod configuration
+so that those pods will have access to it. The MySQL pod will set the
+database password, and the WordPress pod will use the password to
+access the database.
 
-We'll define and start a [service](../../docs/user-guide/services.md) that lets other pods access the mysql database on a known port and host.
-We will specifically name the service `mysql`.  This will let us leverage the support for [Docker-links-compatible](../../docs/user-guide/services.md#how-do-they-work) service environment variables when we set up the wordpress pod. The wordpress Docker image expects to be linked to a mysql container named `mysql`, as you can see in the "How to use this image" section on the wordpress docker hub [page](https://registry.hub.docker.com/_/wordpress/).
+## Deploy MySQL
 
-So if we label our Kubernetes mysql service `mysql`, the wordpress pod will be able to use the Docker-links-compatible environment variables, defined by Kubernetes, to connect to the database.
+Now that the persistent disks and secrets are defined, the Kubernetes
+pods can be launched. Start MySQL using
+[mysql-deployment.yaml](mysql-deployment.yaml).
 
-The [`mysql-service.yaml`](mysql-service.yaml) file looks like this:
-
-<!-- BEGIN MUNGE: EXAMPLE mysql-service.yaml -->
-
-```yaml
-apiVersion: v1
-kind: Service
-metadata: 
-  labels: 
-    name: mysql
-  name: mysql
-spec: 
-  ports:
-    # the port that this service should serve on
-    - port: 3306
-  # label keys and values that must match in order to receive traffic for this service
-  selector: 
-    name: mysql
+```shell
+kubectl create -f $KUBE_REPO/examples/mysql-wordpress-pd/mysql-deployment.yaml
 ```
 
-[Download example](mysql-service.yaml?raw=true)
-<!-- END MUNGE: EXAMPLE mysql-service.yaml -->
-
-Start the service like this:
-
-```sh
-$ kubectl create -f examples/mysql-wordpress-pd/mysql-service.yaml
+Take a look at [mysql-deployment.yaml](mysql-deployment.yaml), and
```

Comment:
You didn't want to auto-include the yaml?
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/how-to-doc.md#how-to-include-an-example

@johndmulhausen may also have a way to do that.